src/checkout/checkout-store-error-selector.ts

```diff
@@ -36,7 +37,7 @@ export default interface CheckoutStoreErrorSelector {
      *
      * @returns The error object if unable to submit, otherwise undefined.
      */
-    getSubmitOrderError(): Error | undefined;
+    getSubmitOrderError(): Error | CartChangedError | undefined;
```

Comment:
this was done in order to get the type `CartChangedError` exported in `checkout-sdk.d.ts`